db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# User.destroy_all
```

Comment:
All the seed info is commented ?

Reply:
I commented a lot of the seeds methods because I didn't know how to code it so it can create a new custom database each time assigning a aircraft to an existing user (or rather, a newly generated user) as well as create a booking between users and aircrafts. So I did each creation step of the database manually and commented the other methods out when running the seed file

Reply:
I'll make some edits and we can chat on Thursday - what you have written is already good just the last booking bit!